### PR TITLE
Modify sample notebook.

### PR DESCRIPTION
**Pull request by qimingj**
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/pipelines/181)

### CODE REVIEW

File: samples/notebooks/KubeFlow Pipeline Using TFX OSS Components.ipynb [hunk 1]
```diff
@@ -32,9 +32,7 @@
    "source": [
     "# Install Pipeline SDK\n",
     "!pip3 install https://storage.googleapis.com/ml-pipeline/release/0.1.1/kfp.tar.gz --upgrade\n",
-    "    \n",
```
Comment (gaoning777) [inline samples/notebooks/KubeFlow Pipeline Using TFX OSS Components.ipynb]: I thought we just added this. Why do we delete the kubernetes installation?

Comment (qimingj) [reply samples/notebooks/KubeFlow Pipeline Using TFX OSS Components.ipynb]: We have that in setup.py now.

**Pull request merged**